src/customer-service/Data/BangazonConnection.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using Microsoft.Data.Sqlite;
+
+namespace customer_service.Data
+{
+    /**
+     * Class: BangazonConnection
+     * Purpose: Connects App with DB allows for executing and reading commands
+     * Author: Matt Hamil/Garrett Vangilder
+     * Methods:
+     *     
+     *     void insert(string query) - This method allows for users to insert new information into a SQLite database
+     *     void execute(string query, Action<SqliteDataReader> handler) - This method opens the database, executes a given command(handler) and then closes the connection
+     */
+    public class BangazonConnection
+    {
+        private string _path = System.Environment.GetEnvironmentVariable("BangazonWeb_Db_Path");
+
+        /**
+         * Purpose: Inserts new information into the datase
+         * Arguments:
+         *     query - database query
+         * Return:
+         *     n/a 
+         */
+        public void insert(string query)
+        {
+            SqliteConnection dbcon = new SqliteConnection(_path);

```

Comment:
Fixed in commit https://github.com/TeamCharles/customer-service/pull/9/commits/7abd15594deb8eb30e62a8a9d06350e4976c1cb3